scipy/spatial/transform/_rotation.pyx

```diff
@@ -2945,6 +2957,9 @@ cdef class Rotation:
         >>> r.mean().as_euler('zyx', degrees=True)
         array([0.24945696, 0.25054542, 0.24945696])
         """
+        if self._quat.shape[0] == 0:
+            raise ValueError("Mean of empty rotation.")
```

Comment:
Perhaps make a more proper sentence like: "Mean of an empty rotation set is undefined" or "Can't compute mean of an empty rotation (set)", something like that.